interfaces/cython/cantera/test/test_thermo.py

```diff
@@ -709,7 +709,9 @@ def test_setHP_highT(self):
         self.assertTrue(self.gas.T > self.gas.max_temp)
 
     def test_volume(self):
-        """ This phase should follow the ideal gas law """
+        """
```

Comment:
I assumed it was something like a doctest. It could be a comment, but I think it doesn't really matter, since some of the other functions also have docstrings.